core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1822,8 +1822,7 @@ object Chunk extends ChunkFactory with ChunkPlatformSpecific {
     implicit val classTag: ClassTag[A] =
       Tags.fromValue(a)
 
-    override val length =
```

Comment:
Any reason to prefer having a `val` here? 🤔

Reply:
Should not be for primitive constants